source/marsGameSound.vwf.yaml

```diff
@@ -20,6 +20,12 @@ properties:
   soundSet:
     - soundName: "musicStandardGameplay"
       soundURL: "assets/sounds/music/musicStandardGameplayLoopFullMix.ogg"
+      useTextToSpeech: true
+      ttsAmplitude: 100
+      ttsVariant: "m1"
+      ttsWordGap: 0
+      ttsSpeed: 175
+      ttsPitch: 50
```

Comment:
Also, add warnings if you specify text-to-speech but no line is given (go off the actual text-to-speech line, not the subtitle), or if a sound file is specified on a text-to-speech-line.

Also, separate out the string that we generate the speech from, instead of using the subtitle.